hack/lib/tracing.bash

```diff
@@ -62,7 +62,7 @@ EOF
 }
 
 function enable_eventing_tracing {
-  header_text "Configuring tracing for Eventing"
+  logger.info "Configuring tracing for Eventing"
```

Comment:
Is this related to this PR?

Reply:
Yup. I can remove that.

Reply:
In fact this is related. This function `header_text` doesn't exist. 

Scripts should have failed on this, but they don't because of our Bash conditional style.